source/getting_started/aiot_intelligent_edge/coco_for_applications/end_user_client_applications/android_end_user_client_applications/quick_start_guide_android_end_user_client_apps.rst

```diff
@@ -3,12 +3,332 @@
 Quick Start Guide
 =================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
+.. sectionauthor:: Krishna
 
-   **Coming Soon**
+This guide provides a quick understanding of API with example code snippets to get started. 
 
-.. sectionauthor:: Krishna
+.. contents::
+  :local:
+
+
+Configuring CocoClient
+----------------------
+
+The base SDK Class that developer will call. CocoClient class follows the singleton design pattern so the configuration goes as follows:
+
+   .. code-block:: java
+   
+        public class SplashActivity extends AppCompatActivity {
+        @Override
+        protected void onCreate(Bundle savedInstanceState) {
+          if (null != CocoClient.getInstance()) {
+            return;
+          }
+   
+          new CocoClient.Configurator()
+              .withCreator(new CreatorEx())
+              .withPlatform(new PlatformInterface() {
+                @Override
+                public String getCwdPath() {
+                  return "provide_current_working_directory";
+                }
+   
+                @Override
+                public String getClientId() {
+                  return "your_client_id_here";
+                }
+   
+                @Override
+                public String getAppAccessList() {
+                  // JSON that describes SDK the capabilities your app will deal
+                  return "{\"appCapabilities\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]}";
+                }
+   
+                @Override
+                public void authCallback(String authorizationEndpoint, String tokenEndpoint) {
+                  // override this callback to perform authentication, This is triggered by SDK when tokens are expired
+                }
+              }).configure();
+        }
+      }
+
+.. important:: 
+
+   ``.withCreator(new CreatorEx())`` is essential as it creates Android
+   flavoured instances of NetworkEx, DeviceEx, ResourceEx, AttributeEx, etc. 
+
+   | Above Ex suffixed classes are sub classes for Network, Device, Resource, Attribute respectively.
+
+----
+
+Authentication
+--------------
+
+1. To login COCO users with their accounts we offer **CocoLoginMgr** a wrapper around **AppAuth-Android** to authenticate users and access their COCO account. The code snippet below achieves Login with COCO using simple result launcher.
+
+   .. code-block:: java
```

Comment:
The Code block here is not appearing in the github preview, whereas the code block for the earlier section "Configuring CocoClient" is appearing in the github preview.

Reply:
Its fine, I verified every code block is rendering properly with emphasized lines, upon sphinx doc generation.